force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -227,19 +286,21 @@ public with sharing class ClassesObjectsMethods {
      */
     public class Book {
         // Declare three public instance variables - title, author, pages
-        // title
-        // author
-        // pages
+        public String title;
+        public String author;
+        public Integer pages;
     
         // Book constructor to initialize the title and author
         public Book() {
             // Initialize the title and author
+            title = 'A Brief History of Time';
+            author = 'Stephen Hawking';
```

Comment:
You hard coded your constructor. It will create the same book every time

Reply:
Agreed.   Probably adding an overloaded constructor will avoid the same book getting created.